topics/ecology/tutorials/phylogeny-data-prep/tutorial.md

```diff
@@ -0,0 +1,339 @@
+In this tutorial we prepare genetic sequence data for phylogenetic reconstruction.
+We will use sequences from chromosome 5 of five strains of the yeast *Saccharomyces cerevisiae*. This requires first the prediction of protein coding genes from the genome. We use Funannotate to predict protein-coding genes . Next, we find the proteins that are present in more than one genome, called orthologs, using Proteinortho, and extract a subset with orthologs that are present in all samples. Finally, each set of orthologs is aligned using ClustalW. The resulting dataset is ready to be used for phylogenetic reconstruction.
+
+
+**If you are starting from sequence reads, please follow
+[An Introduction to Genome Assembly]({{ site.baseurl }}/topics/assembly/tutorials/general-introduction/tutorial.html), and the appropriate genome assembly training for your sequencing technology from GTN's [Assembly]({{ site.baseurl }}/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html) section**
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Get data
+For this training we will use a subset of the genome (chromosome 5) from four strains of *S. cerevisiae*. The GenBank annotated sequenced were produced using 'funannotate predict annotation' (Galaxy Version 1.8.9+galaxy2) on the nucleotide sequences sequences.
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial
+> 2. Import the files from [Zenodo]({{ https://zenodo.org/record/6610704#.Ypn3FzlBw5k }}) or from
+>    the shared data library (`GTN - Material` -> `{{ ecology }}`
+>     -> `{{ phylogenetic-data-prep }}`):
+>
+>    ```
+>    https://zenodo.org/record/6610704/files/BK006939.2.genbank
+>    https://zenodo.org/record/6610704/files/BK006939.2.genome.fasta
+>    https://zenodo.org/record/6610704/files/CM000925.1.genbank
+>    https://zenodo.org/record/6610704/files/CM000925.1.genome.fasta
+>    https://zenodo.org/record/6610704/files/CM005043.2.genbank
+>    https://zenodo.org/record/6610704/files/CM005043.2.genome.fasta
+>    https://zenodo.org/record/6610704/files/CM005299.1.genbank
+>    https://zenodo.org/record/6610704/files/CM005299.1.genome.fasta
+>    ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>
+> 3. Optional: Rename each dataset to its accession number followed by '.genome.fasta' or '.genbank'.
+> 4. Group the datasets into [collections](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/collections/tutorial.html). These will ease data handling and help minimize the clutter in your history. Make a collection of nucleotide sequences and another of protein sequences.
```

Comment:
This upload step could be maybe simplified with the rule-based uploader.

You could provide a table like

```
https://zenodo.org/record/6610704/files/BK006939.2.genbank genbank 
https://zenodo.org/record/6610704/files/BK006939.2.genome.fasta fasta
https://zenodo.org/record/6610704/files/CM000925.1.genbank genbank
https://zenodo.org/record/6610704/files/CM000925.1.genome.fasta fasta
https://zenodo.org/record/6610704/files/CM005043.2.genbank genbank
https://zenodo.org/record/6610704/files/CM005043.2.genome.fasta fasta
https://zenodo.org/record/6610704/files/CM005299.1.genbank genbank
https://zenodo.org/record/6610704/files/CM005299.1.genome.fasta fasta
```

and then have students use the RBU to:

- build a dataset collection from table
- assign column 1 as url
- column 2  as filetype + collection name
- create a new column stripping the base path (it's an option in there somewhere)
- and use that new column 3 as the element identifier.

It's a small tradeoff of learning a little bit of complexity, but maybe it gives students more tools that can scale better for large datasets than manually renaming files / manually creating collections?

Reply:
I definitely see the learning value of doing it this way. But I think, in the end, it makes it harder for the student to get to the analysis part and is a bit beyond the point of this tutorial.

Reply:
Ok, fine for me.